docs/Using_the_EasyBuild_command_line.rst

```diff
@@ -260,6 +260,16 @@ Use ``eb --force/-f`` to force the reinstallation of a given easyconfig/module.
 .. tip:: Combine ``--force`` with ``--dry-run`` to get a good view on which installations will be forced.
    (cfr. :ref:`get_an_overview`)
 
+Rebuild installation, ``--rebuild``
```

Comment:
wouldn't it make sense to collapse the sections for `--force` and `--rebuild` instead? this is more or less a copy-paste, not a big fan of that